plugin/src/test/java/testcode/InsecureRandom.java

```diff
@@ -32,16 +56,26 @@ public static void mathOther() {
         random.nextInt(); //This should not raise any warning
     }
 
+    //True positive PREDICTABLE_RANDOM_SCALA
     public static void scalaRandom() {
         new scala.util.Random();
         new scala.util.Random(new scala.Long());
     }
 
+    //True positive special static nextLong
+    public static void staticNextLong() {
+        System.out.println(JVMRandom.nextLong());
```

Comment:
There is a compilation error on this line. I think it should be something like `new JVMRandom().nextLong()`
```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 31.834 s
[INFO] Finished at: 2018-12-04T14:39:57Z
[INFO] Final Memory: 70M/1061M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:2.3.2:testCompile (default-testCompile) on project findsecbugs-plugin: Compilation failure: Compilation failure: 
[ERROR] /home/travis/build/find-sec-bugs/find-sec-bugs/plugin/src/test/java/com/h3xstream/findsecbugs/PredictableRandomDetectorTest.java:[101,24] error: no suitable method found for doReportBug(BugInstanceMatcherBuilder)
[ERROR] /home/travis/build/find-sec-bugs/find-sec-bugs/plugin/src/test/java/testcode/InsecureRandom.java:[67,36] error: non-static method nextLong() cannot be referenced from a static context
```

Reply:
My bad, i missed that static version of nextLong takes one argument and it is the only difference between static and non-static functions.  It should be JVMRandom.nextLong(5) for example.